bindings/python/test/access/test_visibility_criterion.py

```
@@ -0,0 +1,199 @@
+# Apache License 2.0
+
+import pytest
+import numpy as np
+
+from ostk.mathematics.object import RealInterval
+
+from ostk.physics import Environment
+from ostk.physics.time import Instant
+from ostk.physics.time import DateTime
+from ostk.physics.time import Scale
+from ostk.physics.coordinate.spherical import AER
```

Ruff (0.8.2)

`ostk.physics.coordinate.spherical.AER` imported but unused

Remove unused import: `ostk.physics.coordinate.spherical.AER`

(F401)